agr_java_core/src/main/java/org/alliancegenome/es/index/site/cache/IndexerCache.java

```diff
@@ -62,6 +62,17 @@
         String id = document.getPrimaryKey();
 
         document.setAlleles(alleles.get(id));
+
+        Allele a= alleleMap.get(id);
+        if(a!=null) {
+            if(diseases != null && diseases.get(id)!=null )
+                a.setDisease(diseases.get(id).size() > 0);
+            if(phenotypeStatements != null && phenotypeStatements.get(id)!=null)
+                a.setPhenotype(phenotypeStatements.get(id).size() > 0);
+
+        }
+        if(document instanceof AlleleVariantSequence)
+        ((AlleleVariantSequence) document).setAllele(a);
```

Comment:
@jt15 this code exists in the AlleleIndexerCache.java so this would be duplicate code.

Reply:
its not duplicated

Reply:
Please take a look at: `agr_java_core/src/main/java/org/alliancegenome/es/index/site/cache/AlleleDocumentCache.java`

Reply:
updated and is working fine